Add blog post: HowTo create a image template

bgeuken opened the pull request.

bgeuken commented:
Current release date would be 2017-02-27

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+---
+
+Do you consider to write your own image templates? But you don't know
+how to start? You have come to the right place.
```
I think _**You are in the right place!**_ sounds better :thinking:

Ana06 replied:
> i don't really see what the point of these two lines above are?

to make the blog post gripping! I like them :heart_eyes_cat:

evanrolfe replied:
If it were me I wouldn't include these lines but I don't mind if you prefer to keep it @bgeuken maybe I'm just a boring doc writer ;)

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+category: documentation
+---
+
+Do you consider to write your own image templates? But you don't know
```
Isn't this only one question? I mean:

_**Do you consider to write your own image templates but you don't know how to start?**_

evanrolfe replied:
Should be "Have you considered writing your own templates template but don't know how to start?"

evanrolfe replied:
@bgeuken i think this still hasn't been changed?

bgeuken replied:
@evanrolfe Hm, I thought both versions were okay. "Have you considered" sounds to me like it's about something in the past. Isn't "Do you consider" (right now) more fitting?

evanrolfe replied:
Thats true it makes more sense to be in the present tense, but "Do you consider" is definitely wrong though. I can't explain why though to be fair!

You could say "Would you consider...", "Are you considering...", "Would you like to...", "Do you want to..."?

bgeuken replied:
I'll take "Are you considering" for now

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+
+<img src="/images/posts/image_templates_project.png" style="margin:20px;" title="Creating a Subproject">
+
+This will be the place where you can collect your image templates.
```
doesn't _**can** collect_ sound strange? What about _**will** collect_? or _**will be able to** collect_?

bgeuken replied:
Maybe "This will be the place where you collect your image templates." ?

Ana06 replied:
:+1:

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+
+Go to the 'Repositories' tab, click on 'Add repositories' and check the 'KIWI image build' checkbox.
+
+Now the only thing missing are some source files of your image configuration. If you don't know
```
_Now the only thing missing **is** to add some source files of your image configuration. If you don't know_

ChrisBr replied:
@evanrolfe we need some native speaker here :) is vs. are?

evanrolfe replied:
Ana is right ;)

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+
+## Publish your own image templates on the templates page
+
+You might want to provide your image configuration to other users. In that case either
```
either? :confused:

Ana06 commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+<img src="/images/posts/image_templates_submit_request.png" style="margin:20px;" title="Image Template Submit Request">
+
+Or send a mail to the [OBS admins](admin@opensuse.org) and request to have your
+templates project added to the templates page.
```
I think this sound better:

_...and request that your template project **is** added to the templates page._

bgeuken replied:
But I keep naming it templates project to emphasize that one adds the whole project there. Which can have multiple image templates.

Ana06 replied:
Then

...and request that your templates project **is** added to the templates page.

:slightly_smiling_face:

evanrolfe commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+how to start? You have come to the right place.
+
+In this short article you will learn to create you own image templates,
+... and publish them.
```
Again this seems like unnecessary text? Maybe change the title to "Creating and Publishing Image Templates" then you can get rid of this sentence?

evanrolfe commented on _posts/documentation/2017-01-19-creating-your-own-image-template.md:
```diff
@@ -0,0 +1,51 @@
+
+<img src="/images/posts/image_templates_submit_request.png" style="margin:20px;" title="Image Template Submit Request">
+
+Or send a mail to the [OBS admins](admin@opensuse.org) and request to have your
```
"send an email"

bgeuken commented:
\*Updated & rebased \*

I've removed the duplicated blog article, included your comments and added some more text to the part of submitting changes back to upstream projects

evanrolfe commented:
@bgeuken it looks like there are still some comments that haven't been implemented or addressed?
